### DISCUSSION TITLE
SMS Gateway API Connect Request

### DISCUSSION THREAD

**Discussion by thisirasandawath** [Blair2004/NexoPOS#534] [General]
Please set SMS Utility support to custom SMS Gateway Get/Post method. I need to integrate our custom SMS Gateway API. Can anyone support to me? Twilio, Click Send, Telnyx Most popular brands but their prices are higher for our country. I need to connect our SMS Gateway System to this. Please Set up.

**Comment by smsala** [comment]
Hello @thisirasandawath 

Our SMSala Dev or Support Team will provide SMS Gateway API integration

Get More Info visit our website [www.smsala.com](https://smsala.com)

**Comment by Blair2004** [comment]
Hi @thisirasandawath i'll write shortly a guide for integrating your own SMS gateway for the SMS utility module. 
Thank you for pointing that concern.

**Reply by thisirasandawath** [reply]
Thank you @Blair2004 I'm waiting. I send my demo SMS login details

**Comment by thisirasandawath** [comment]
Hello Team,

*Our SMS Gateway Login Credentials*
Login: https://smsgateway.greenline.lk/
Username: ***@***.***
Password: ***@***.***

Developers Docs: https://smsgateway.greenline.lk/developers
Parameters: SenderID / API Token
Demo Sender ID: *Demo Green*
API Token: *24|Tnj7ci4RG8y8aqqbMdRKKYBvfVBgoX5wkhww88WY*
Test Our Number: 94755399599 (Use to send SMS)

Thank you. Best Regards.

*Thisira Sandawath*

------------------------------

**This e-mail is confidential. It may also be legally privileged. If you
are not the intended recipient or have received it in error, please delete
it and all copies from your system and notify the sender immediately by
return e-mail. Any unauthorized reading, reproducing, printing, or further
dissemination of this e-mail or its contents is strictly prohibited and may
be unlawful. Internet communications cannot be guaranteed to be timely,
secure, error, or virus-free. The sender does not accept liability for any
errors or omissions.
